crates/subspace-networking/examples/get-peers-complex.rs

```diff
@@ -90,12 +91,22 @@ async fn main() {
 
     tokio::time::sleep(Duration::from_secs(1)).await;
 
-    let peers = node
-        .get_closest_peers(expected_node_id.into())
-        .await
-        .unwrap();
+    // Prepare multihash to look for in Kademlia
+    let encoding = expected_node_id.as_ref().digest();
+    let public_key = identity::PublicKey::from_protobuf_encoding(encoding)
+        .expect("Invalid public key from PeerId.");
+    let peer_id_public_key = if let identity::PublicKey::Sr25519(pk) = public_key {
+        pk.encode()
+    } else {
+        panic!("Expected PublicKey::Sr25519")
+    };
+
+    let key = Code::Identity.digest(&peer_id_public_key);
```

Comment:
It would be nicer to generate an identity explicitly and grab public key from there, but I guess this also works :upside_down_face:

Reply:
Ok. I'll address it in the following PR as well.